sktime/forecasting/model_selection/tests/test_split.py

```diff
@@ -325,6 +325,28 @@ def test_sliding_window_splitter_start_with_empty_window(
             )
 
 
+@pytest.mark.parametrize("y", TEST_YS)
+@pytest.mark.parametrize("fh", [*TEST_FHS, *TEST_FHS_TIMEDELTA])
+@pytest.mark.parametrize("step_length", TEST_STEP_LENGTHS)
+def test_expanding_window_splitter_start_with_initial_window_zero(y, fh, step_length):
+    """Test ExpandingWindowSplitter."""
+    if _inputs_are_supported([fh, step_length, 0]):
```

Comment:
please, create a variable `initial_window = 0` above `if` statement and use it instead of zeros below

Reply:
done